### ISSUE TITLE
Crash after like 6 minutes

### ISSUE THREAD

**Issue by Exaint** [Kaldaien/FAR#138]
Hey using the newest verion of far 

and the game is crashing every few minutes ...crash log is attached
[crash.log](https://github.com/Kaldaien/FAR/files/2475639/crash.log)

if u dont want to click:

10/13/2018 18:26:28.371: -----------------------------------------------------------
10/13/2018 18:26:28.372: [! Except !] 	<< EXCEPTION_ACCESS_VIOLATION >>
10/13/2018 18:26:28.372: -----------------------------------------------------------
10/13/2018 18:26:28.372: [ FaultMod ]  # File.....: 'D:\Program Files (x86)\Steam\steamapps\common\NieRAutomata\NieRAutomata.exe'
10/13/2018 18:26:28.372: [ FaultMod ]  * RIP Addr.: NieRAutomata.exe+000000000092373Eh
10/13/2018 18:26:28.372: [StackFrame] <-> Rip=00014092373eh, Rsp=00000fd4fa70h, Rbp=000000008004h
10/13/2018 18:26:28.372: [StackFrame] >-< Rsi=00000000002fh, Rdi=00002306ebd0h
10/13/2018 18:26:28.372: [  GP Reg  ]       rax:     0x00000fd4fac8
10/13/2018 18:26:28.372: [  GP Reg  ]       rbx:     0x000000002050
10/13/2018 18:26:28.372: [  GP Reg  ]       rcx:     0x000022e84eb0
10/13/2018 18:26:28.372: [  GP Reg  ]       rdx:     0x00000fd4fb00
10/13/2018 18:26:28.372: [  GP Reg  ]        r8:     0x00004e9c16f4       r9:      0x000022e84f10
10/13/2018 18:26:28.372: [  GP Reg  ]       r10:     0x000022e84eb0      r11:      0x00004e9c16f4
10/13/2018 18:26:28.372: [  GP Reg  ]       r12:     0x000000000001      r13:      0x000140000000
10/13/2018 18:26:28.372: [  GP Reg  ]       r14:     0x000000004002      r15:      0x000000000000
10/13/2018 18:26:28.372: [ GP Flags ]       EFlags:  0x00010202
10/13/2018 18:26:28.372: -----------------------------------------------------------
10/13/2018 18:26:28.373: -----------------------------------------------------------
10/13/2018 18:26:28.374: -----------------------------------------------------------
10/13/2018 18:26:28.374: [! Except !] 	<< EXCEPTION_ACCESS_VIOLATION >>
10/13/2018 18:26:28.374: -----------------------------------------------------------
10/13/2018 18:26:28.374: [ FaultMod ]  # File.....: 'D:\Program Files (x86)\Steam\steamapps\common\NieRAutomata\NieRAutomata.exe'
10/13/2018 18:26:28.374: [ FaultMod ]  * RIP Addr.: NieRAutomata.exe+000000000092373Eh
10/13/2018 18:26:28.374: [StackFrame] <-> Rip=00014092373eh, Rsp=00000fd4fa70h, Rbp=000000008004h
10/13/2018 18:26:28.374: [StackFrame] >-< Rsi=00000000002fh, Rdi=00002306ebd0h
10/13/2018 18:26:28.374: [  GP Reg  ]       rax:     0x00000fd4fac8
10/13/2018 18:26:28.374: [  GP Reg  ]       rbx:     0x000000002050
10/13/2018 18:26:28.374: [  GP Reg  ]       rcx:     0x000022e84eb0
10/13/2018 18:26:28.374: [  GP Reg  ]       rdx:     0x00000fd4fb00
10/13/2018 18:26:28.374: [  GP Reg  ]        r8:     0x00004e9c16f4       r9:      0x000022e84f10
10/13/2018 18:26:28.374: [  GP Reg  ]       r10:     0x000022e84eb0      r11:      0x00004e9c16f4
10/13/2018 18:26:28.374: [  GP Reg  ]       r12:     0x000000000001      r13:      0x000140000000
10/13/2018 18:26:28.374: [  GP Reg  ]       r14:     0x000000004002      r15:      0x000000000000
10/13/2018 18:26:28.374: [ GP Flags ]       EFlags:  0x00010202
10/13/2018 18:26:28.374: -----------------------------------------------------------
10/13/2018 18:26:28.374: -----------------------------------------------------------
10/13/2018 18:26:28.375: -----------------------------------------------------------
10/13/2018 18:26:28.375: [! Except !] 	<< EXCEPTION_ACCESS_VIOLATION >>
10/13/2018 18:26:28.375: -----------------------------------------------------------
10/13/2018 18:26:28.375: [ FaultMod ]  # File.....: 'D:\Program Files (x86)\Steam\steamapps\common\NieRAutomata\NieRAutomata.exe'
10/13/2018 18:26:28.375: [ FaultMod ]  * RIP Addr.: NieRAutomata.exe+000000000092373Eh
10/13/2018 18:26:28.375: [StackFrame] <-> Rip=00014092373eh, Rsp=00000fd4fa70h, Rbp=000000008004h
10/13/2018 18:26:28.375: [StackFrame] >-< Rsi=00000000002fh, Rdi=00002306ebd0h
10/13/2018 18:26:28.375: [  GP Reg  ]       rax:     0x00000fd4fac8
10/13/2018 18:26:28.375: [  GP Reg  ]       rbx:     0x000000002050
10/13/2018 18:26:28.375: [  GP Reg  ]       rcx:     0x000022e84eb0
10/13/2018 18:26:28.375: [  GP Reg  ]       rdx:     0x00000fd4fb00
10/13/2018 18:26:28.375: [  GP Reg  ]        r8:     0x00004e9c16f4       r9:      0x000022e84f10
10/13/2018 18:26:28.375: [  GP Reg  ]       r10:     0x000022e84eb0      r11:      0x00004e9c16f4
10/13/2018 18:26:28.375: [  GP Reg  ]       r12:     0x000000000001      r13:      0x000140000000
10/13/2018 18:26:28.375: [  GP Reg  ]       r14:     0x000000004002      r15:      0x000000000000
10/13/2018 18:26:28.375: [ GP Flags ]       EFlags:  0x00010202
10/13/2018 18:26:28.375: -----------------------------------------------------------
10/13/2018 18:26:28.376: -----------------------------------------------------------

**Comment by Innsomatic** [comment]
Can confirm same issue for myself -- Vanilla game runs ...engh.  With FAR mod game will crash after *about* the same timeframe.   Just deleted my directory, will post additional log after *this* reinstall (done this once already).

**Comment by Innsomatic** [comment]
So In response to above. I reinstalled, both Nier and Far (Hard deleted -- files, some registry entries, everything by a save data file which I backed up, 

I was able to get maybe about an hour before it crashed.   Tried again, crashed almost instantly.  Same result until I restart comp.  
Another funny issue is:  Windows 10 had the notification: "display driver failed to start" and my steam was shut down right after the crash, which usually isn't the case.   I have a GTX 1080, with UTD drivers from this month.   I even ran the UDD and reinstalled just to make sure something wasn't messed up.

Today, I tried to disable all Third party items I could,  thinking maybe the Steam API is causing the issue (since it crashed too).  I upped the logging level mentioned in FAR, but I don't it didn't do much.  Nothing in my Crash log, and the folder I have is from last night,  but there's no error or exceptions in the log.  Most logs seem to come from yesterday.  Got a funky exception window this time though filled with non-sense characters (about 25-30) but nothing in logs telling me what it meant.   -- I can post but at this point I don't think there's much renewed interest in fixing this beast of a port.  I'll keep monitoring -- updating this issue accordingly if I have any breakthroughs.
